MOBILE_CHECKOUT_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,190 @@
+# PayPal Mobile Checkout SDK: Migration Guide
+
+This guide outlines how to update your integration from using the soon-to-be-deprecated [PayPal Mobile Checkout SDK](https://developer.paypal.com/limited-release/paypal-mobile-checkout/) to the new PayPal Mobile [iOS SDK](https://github.com/paypal/iOS-SDK).
+
+## Pre-Requisites
+In order to use this migration guide, you must:
+
+1. Have a server-side integration with the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2/). Please update to Orders v2 if you're on [Payments V1](https://developer.paypal.com/docs/api/payments/v1/) or [NVP/SOAP](https://developer.paypal.com/api/nvp-soap/).
+1. Enable your server to [fetch an Access Token](https://developer.paypal.com/reference/get-an-access-token/).
+1. Enable your server to create an [Order ID](https://developer.paypal.com/docs/api/orders/v2/).
+1. Enable your server to [PATCH](https://developer.paypal.com/docs/api/orders/v2/#orders_patch) an order.
+    * _Note:_ This is **only required** if you create your order ID with [`shipping_preference`](https://developer.paypal.com/docs/api/orders/v2/#definition-order_application_context) = `GET_FROM_FILE`. See step 6 in the guides below.
+
+## Client-Side
+
+*Assuming the pre-requisites are met, this migration should take ~1 developer day to complete.*
+
+
+1. Add the new SDK to your app
```

Comment:
I wonder if calling this the "new SDK" will make people hesitant like it's untested/not final. Maybe just saying "Add the PayPal iOS SDK to your app" would be better? Very take it or leave it though.

Reply:
Yeh that's a good call. IMO the names of the SDKs are so similar it is hard to distinguish them, so I opted for "new".